binary_search/BinarySearch.java

```diff
@@ -0,0 +1,34 @@
+public class BinarySearch {
+
+    static int binarySearch(int[] arr, int arrSize, int searchElement) {
```

Comment:
Not necessary! Yeah! it is redundant.I just removed.
But why codeclimate failed.